js/source/image_source.js

```diff
@@ -156,6 +156,8 @@ class ImageSource extends Evented {
             gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.LINEAR);
             gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
             gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, image);
+            this.map.animationLoop.set(this.tile.timeAdded + this.map.style.rasterFadeDuration - Date.now());
```

Comment:
Why did animation loop work for raster fades before adding this line? Do we set the loop in several places for this?